### PR TITLE
added password hashing

### PR DESCRIPTION
**Pull request by MaliukDaria**

### CODE REVIEW

**Review by Borisko130 — APPROVED** [review]

File: src/main/resources/init_db.sql [hunk 1]
```diff
@@ -131,18 +131,9 @@ ALTER TABLE `online_store`.`users_roles`
...
-INSERT INTO orders (user_id) VALUES (1);
-INSERT INTO orders (user_id) VALUES (2);
-INSERT INTO orders (user_id) VALUES (3);
-
 ALTER TABLE `online_store`.`users`
     DROP INDEX `login_UNIQUE` ;
 ;
+
+ALTER TABLE `online_store`.`users`
+    ADD COLUMN `salt` VARBINARY(255) NOT NULL AFTER `isDeleted`;
```
Comment (dmytro-hryhoruk) [inline src/main/resources/init_db.sql]: let's modify the original create table statement

File: src/main/webapp/WEB-INF/views/orders/allOrders.jsp [hunk 2]
```diff
@@ -5,6 +5,38 @@
...
+            padding: 14px 20px;
+            text-decoration: none;
+        }
+        .navbar a.right {
+            float: right;
+        }
+        .navbar a:hover {
+            background-color: #ddd;
+            color: #4c4c4c;
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/orders/allOrders.jsp]: can we move this  to separate file?

File: src/main/webapp/WEB-INF/views/orders/userOrders.jsp [hunk 3]
```diff
@@ -5,6 +5,38 @@
...
+            padding: 14px 20px;
+            text-decoration: none;
+        }
+        .navbar a.right {
+            float: right;
+        }
+        .navbar a:hover {
+            background-color: #ddd;
+            color: #4c4c4c;
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/orders/userOrders.jsp]: same, especially  if this the same style

File: src/main/webapp/WEB-INF/views/products/all.jsp [hunk 4]
```diff
@@ -6,6 +6,38 @@
...
+            padding: 14px 20px;
+            text-decoration: none;
+        }
+        .navbar a.right {
+            float: right;
+        }
+        .navbar a:hover {
+            background-color: #ddd;
+            color: #4c4c4c;
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/products/all.jsp]: and here

File: src/main/webapp/WEB-INF/views/products/userCatalog.jsp [hunk 5]
```diff
@@ -6,6 +6,38 @@
...
+            padding: 14px 20px;
+            text-decoration: none;
+        }
+        .navbar a.right {
+            float: right;
+        }
+        .navbar a:hover {
+            background-color: #ddd;
+            color: #4c4c4c;
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/products/userCatalog.jsp]: +

File: src/main/webapp/WEB-INF/views/shoppingcart/products.jsp [hunk 6]
```diff
@@ -5,6 +5,38 @@
...
+            padding: 14px 20px;
+            text-decoration: none;
+        }
+        .navbar a.right {
+            float: right;
+        }
+        .navbar a:hover {
+            background-color: #ddd;
+            color: #4c4c4c;
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/shoppingcart/products.jsp]: and so on and so on

File: src/main/resources/init_db.sql (hunk 1 above)
Comment (MaliukDaria) [reply src/main/resources/init_db.sql]: done

File: src/main/webapp/WEB-INF/views/orders/allOrders.jsp (hunk 2 above)
Comment (MaliukDaria) [reply src/main/webapp/WEB-INF/views/orders/allOrders.jsp]: done

File: src/main/webapp/WEB-INF/views/orders/userOrders.jsp (hunk 3 above)
Comment (MaliukDaria) [reply src/main/webapp/WEB-INF/views/orders/userOrders.jsp]: done

File: src/main/webapp/WEB-INF/views/products/all.jsp (hunk 4 above)
Comment (MaliukDaria) [reply src/main/webapp/WEB-INF/views/products/all.jsp]: done

File: src/main/webapp/WEB-INF/views/products/userCatalog.jsp (hunk 5 above)
Comment (MaliukDaria) [reply src/main/webapp/WEB-INF/views/products/userCatalog.jsp]: done

File: src/main/webapp/WEB-INF/views/shoppingcart/products.jsp (hunk 6 above)
Comment (MaliukDaria) [reply src/main/webapp/WEB-INF/views/shoppingcart/products.jsp]: done

**Review by dmytro-hryhoruk — COMMENTED** [review]
Still i can see a lot of information in <style> tag. Let's try to move it to separate .css files and import here. Useful information how to import them later in your jsp( you can do this even with header.jsp) 
https://stackoverflow.com/questions/7382607/how-to-load-css-file-into-jsp.
And check if you are using the same style in different files so you will not create a duplication. If it mainly the same but has a small difference, then you can add them  with tag <style>

File: src/main/webapp/WEB-INF/views/styles/button.css [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+    background-color: #34432f;
+    color: white;
+    padding: 16px 20px;
+    margin: 8px 0;
+    border: none;
+    cursor: pointer;
+    width: 50%;
+    opacity: 0.9;
+    float: left;
+}
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/styles/button.css]: empty line

File: src/main/webapp/WEB-INF/views/header/header.jsp [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+            overflow: hidden;
+            background-color: #34432f;
+        }
+        .navbar a {
+            float: left;
+            display: block;
+            color: white;
+            text-align: center;
+            padding: 14px 20px;
+            text-decoration: none;
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/header/header.jsp]: even though it's header, let's move its style to separate file too

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/main/webapp/WEB-INF/views/styles/button.css (hunk 7 above)
Comment (MaliukDaria) [reply src/main/webapp/WEB-INF/views/styles/button.css]: done

File: src/main/webapp/WEB-INF/views/header/header.jsp (hunk 8 above)
Comment (MaliukDaria) [reply src/main/webapp/WEB-INF/views/header/header.jsp]: done

**Pull request merged**